packages/@okta/vuepress-site/guides/sign-into-spa/user-info/index.md

```diff
@@ -1,10 +1,13 @@
 ---
 title: Get Info About the User
 ---
-Okta sends back user profile information that you can use in your application.
+
+When a user signs in, their profile information (stored in Okta) is made available to your application. It's common to use this info to update your app's UI.
+
+By default, the profile items (called "claims") Okta returns include the user's email address, first name, and last name. 
```

Comment:
By default, the profile items (called "claims") that Okta returns include the user's email address, first name, and last name. 

adding "that" for translation